### ISSUE TITLE
Can't seem to get denormalization to work

### ISSUE THREAD

**Issue by arlen-yu** [cult-of-coders/grapher#278]
Grapher version 1.3.7. Right now I have the following setup:

```js
// imports/startup/server/main.js
import { migrate } from 'meteor/herteby:denormalize'

console.log('Starting server...')

Meteor.startup(() => {
  migrate('messages', 'senderCache')
})
```

```js
Messages.addLinks({
  sender: {
    type: 'one',
    collection: Users,
    field: 'senderId',
    index: true,
    denormalize: {
      field: 'senderCache',
      body: {
        firstName: 1
      }
    }
  }
})
```

```js
// the query (used with grapher-react)

export default createQuery({
  messages: {
    $filters: {
      senderCache: {
        $exists: true
      }
    }
  }
})
```

But it doesn't seem like the senderCache is appearing since no data appears. Also, the migration seems to have worked based on the server log `migrated senderCache of 4 docs in messages. It took 20ms`.  Am I missing anything? Thanks!

**Comment by arlen-yu** [comment]
Hi @theodorDiaconu. I created a test repo [here](https://github.com/arlen-yu/grapher-boilerplate-denormalization) based off the grapher-boilerplate. Check my latest commit for the changes. 

I tried to add some skeletal changes just to get denormalization up, but I get the following error: 

`Error: After filtering out keys not in the schema, your modifier is now empty`

Maybe I'm missing a crucial step... (tired). Thanks for the help on this and your work on the package.
